src/client/eppo-precomputed-client.ts

```diff
@@ -43,6 +45,15 @@ export type PrecomputedFlagsRequestParameters = {
   skipInitialPoll?: boolean;
 };
 
+export function convertContextAttributesToSubjectAttributes(
+  contextAttributes: ContextAttributes,
+): Attributes {
+  return {
+    ...(contextAttributes.numericAttributes || {}),
+    ...(contextAttributes.categoricalAttributes || {}),
```

Comment:
Test data updated in the following PR to these attribute keys

https://github.com/Eppo-exp/sdk-test-data/pull/94